### DISCUSSION TITLE
Upper limit and lower limit charging for Type C

### DISCUSSION THREAD

**Discussion by cpbrr** [seerge/g-helper#2014] [Ideas]
Might seem abit stupid but it would be nice to have for type c charging since type c charging constantly stops charging and starts charging when it hits charging limit. If you could add a manual button to toggle it on for type c charging or let the computer detect that it's charging through type c that would be great. Thanks!

**Comment by seerge** [comment]
@Skezsy hello. That's not really doable. 

Charge limit is just a single value in percentage that can be set. Then BIOS will just stop charging device as soon as that percentage is reached.

App doesn't monitor / stop / start charge anyhow and can't do that :)
